website/templates/project/registration_editor_templates.mako

```diff
@@ -69,7 +69,7 @@
           <br />
           <div class="row">
             <div class="col-md-12">
-              <div class="form-group" data-bind="css: {has-success: $data.isComplete}">
+              <div class="form-group" data-bind="css: {has-success: $data.value.isValid}">
```

Comment:
Note to self: test validation (eg should this be `isValid()` to get the value of the observable?)
http://knockoutjs.com/documentation/css-binding.html